### ISSUE TITLE
Keyboard focus lost while scrolling timeline via keyboard

### ISSUE THREAD

**Issue by spate** [tootsuite/mastodon#10991]
### Expected behaviour

Keyboard focus should be retained by the timeline while scrolling through the timeline.

### Actual behaviour

Keyboard focus is stolen by an unknown element, and keyboard events stop updating the timeline.

### Steps to reproduce the problem

1. In basic timeline web view, click in whitespace around any toot.
2. Use 'Page down' key to repeatedly scroll down through the timeline until focus lost.

### Specifications

Seen with Mastodon v2.8.4 (mastodon.social)

Browser: Chrome 75.0.3770.80
OS: Windows 10
